File: Binary Search I/python/Day 9/double_exists.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        '''
        Simple bruteforce search to get pairs
        Still figuring out how to apply binary search 🤔
        '''
        for i in range(len(arr)):
          for j in range(len(arr)):
              if(i!=j and arr[i]==2*arr[j]):
                  return True
        return False

File: Binary Search I/cpp/Day 9/DoubleExists.cpp
class Solution {
public:
    bool checkIfExist(vector<int>& arr) {
        /**
        Simple bruteforce search to get pairs
        Still figuring out how to apply binary search 🤔
         */
        for(int i=0;i<arr.size();i++){
          for(int j=0;j<arr.size();j++){
              if(i!=j&&arr[i]==2*arr[j]){
                  return true;
              }
          }
      }
      return false;
    }
};